lib/tasks/admin.rake

```diff
@@ -54,4 +54,12 @@ namespace :admin do
   task openai: :environment do
     Openai.call
   end
+
```

Comment:
@patanj101 - this is how Dan has setup testing for the integrations. Can you note this and setup something similar for the end-to-end process with his help so that we don't need to re-seed every time to test.